server/conf/i18n/messages

```diff
@@ -65,6 +65,20 @@ toast.localeNotSupported=We''re sorry, this program is not fully translated to y
 link.allDone=I''m all done
 link.applyToAnotherProgram=Apply to another program
 link.createAccountOrSignIn=Create account or sign in
+# The title of a pop-up informing the user that they tried to go to the review page but there were errors in the information that they inputted
+modal.errorSaving.review.title=Questions on this page are not complete. Would you still like to leave and begin reviewing?
+# The title of a pop-up informing the user that they tried to go to the previous page but there were errors in the information that they inputted
+modal.errorSaving.previous.title=Questions on this page are not complete. Would you still like to leave and go to the previous page?
+# A message informing the user that there were errors in the information that they inputted. This message also asks the user if they want to (1) go back and fix the errors or (2) discard the information they've inputted and continue to the application review page.
+modal.errorSaving.review.content=There are some errors with the information you''ve filled in. Would you like to go back and fix the errors, or go to the review page without saving your answers?
+# A message informing the user that there were errors in the information that they inputted. This message also asks the user if they want to (1) go back and fix the errors or (2) discard the information they've inputted and continue to the previous page of the application.
+modal.errorSaving.previous.content=There are some errors with the information you''ve filled in. Would you like to go back and fix the errors, or go to the previous question page without saving your answers?
+# Text for a button. When the button is clicked, then all the information the user has inputted will be discarded and they will be taken to the application review page.
+modal.errorSaving.review.noSaveButton=Go to review page without saving
+# Text for a button. When the button is clicked, then all the information the user has inputted will be discarded and they will be taken to the previous page in the application.
+modal.errorSaving.previous.noSaveButton=Go to previous page without saving
+# Text for a button. When the button is clicked, the user will be shown the information that they previously inputted and will be asked to fix the errors with it.
+modal.errorSaving.goBackAndEdit=Go back and fix
```

Comment:
**suggestion:** Maybe name `goBackAndEditButton` for consistency with other buttons?

Reply:
Good idea, done.